src/OpenTracing/StartSpanOptions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+
+namespace OpenTracing
+{
+    /// <summary>
+    /// Provides a way to configure the behavior of <see cref="ITracer.StartSpan"/>.
+    /// </summary>
+    public class StartSpanOptions
```

Comment:
I see your point. Agree that the API footprint should be minimized. If anything should be cut out it is the SpanBuilder. Compared to the Java version the SpanBuilder is more of a SpanStartOptionBuilder, no logic, just a fluent way to add options. After collecting all the parameters instead of building the SpanStartOption like normal builder, it directly calls Tracer.StartSpan. I see it more like the predefined tags, it is just a convenient method to help guide the user. If it is stopping this PR we cut it out.